util/runtime/src/lib.rs

```diff
@@ -54,15 +57,20 @@ impl Handle {
 pub fn new_global_runtime() -> (Handle, StopHandler<()>) {
     let runtime = Builder::new_multi_thread()
         .enable_all()
-        .thread_name("ckb-global-runtime")
+        .thread_name("GlobalRt")
+        .thread_name_fn(|| {
+            static ATOMIC_ID: AtomicUsize = AtomicUsize::new(0);
+            let id = ATOMIC_ID.fetch_add(1, Ordering::SeqCst);
```

Comment:
use `fetch_update` instead, our worker may up and down every time because we use `block in place` on lots of time

Reply:
> `pub fn fetch_add(&self, val: usize, order: Ordering) -> usize`
>
> [This operation wraps around on overflow.](https://doc.rust-lang.org/std/sync/atomic/struct.AtomicUsize.html#method.fetch_add)

But I also will update this code, because if id more than 1_000_000, we still missing the unique id, due to the length limit.